Bool move_node_rdy